docs/CHANGELOG.asciidoc

```diff
@@ -99,6 +100,392 @@ Review important information about the {kib} 8.x releases.
+
+Management::
+* Adds autocomplete suggestions for {esql} queries in Console ({kibana-pull}219980[#219980]).
+Search::
+* Adds a section that enables adding, updating, and deleting query rules written on top of the Query Rules APIs to pin or exclude documents according to criteria you set ({kibana-pull}227226[#227226]).
+
+For more information about the features introduced in 8.19.0, refer to <<whats-new,What's new in 8.19>>.
+
+[[enhancements-and-bug-fixes-v8.19.0]]
+=== Enhancements and fixes
+
+For detailed information about the 8.19.0 release, review the enhancements and fixes.
+
+[float]
+[[enhancement-v8.19.0]]
+==== Enhancements
+Alerting::
+* Ensures the **Reporting** page only shows reports generated in the current space ({kibana-pull}221375[#221375]).
+* Moves the rule settings to a flyout ({kibana-pull}216162[#216162]).
+* Allows users to delete a snooze schedule from a rule using schedule ID ({kibana-pull}213247[#213247]).
+* Allows users to create a snooze schedule for rules using the schedule API ({kibana-pull}210584[#210584]).
+* Implements functionality to add observables, procedures, and custom fields to alerts for {hive} ({kibana-pull}207255[#207255]).
+Dashboards and Visualizations::
+* Updates time-based charts to use the multi-layer time axis by default, providing a better time window context and improved label positioning. ({kibana-pull}210579[#210579]).
+* Adds a **Point visibility** option to Area and Line charts in **Lens** ({kibana-pull}222187[#222187]).
+* Adds a setting to control the data table density in **Lens** ({kibana-pull}220252[#220252]).
+* When clicking "Open in Lens" from a visualization, **Lens** now opens in the same tab ({kibana-pull}217528[#217528]).
+* Adds the ability to open links from **Vega** visualizations in a new tab by specifying the `"target": "_blank"` option for the `usermeta.embedOptions.loader` property of the Vega chart configuration ({kibana-pull}216200[#216200]).
+* Adds globe projection improvements to **Maps** ({kibana-pull}212437[#212437]).
+* Keeps the chart configuration when possible after editing the {esql} visualization's query ({kibana-pull}210780[#210780]).
+* Allows the creation of dynamic aggregations controls for {esql} visualizations ({kibana-pull}210170[#210170]).
+Data ingestion and Fleet::
+* Reuses shared integration policies when duplicating agent policies ({kibana-pull}217872[#217872]).
+* Adds support for collapsible sections in integration overview pages ({kibana-pull}223916[#223916]).
+Discover::
+* Expands the {esql} editor to fit the query size automatically when loading **Discover** ({kibana-pull}225509[#225509]).
+* Shows list of keyboard shortcuts at the bottom of the editor ({kibana-pull}221331[#221331]).
+* Adds suggestions for full text search when writing queries ({kibana-pull}221239[#221239]).
+* Adds full text search suggestions to `STATS ... WHERE` queries ({kibana-pull}220691[#220691]).
```

Comment:
The:

- Shows list of keyboard shortcuts at the bottom of the editor
- dds suggestions for full text search when writing queries 
- Adds full text search suggestions to `STATS ... WHERE` queries

need to go to an ES|QL section and not under Discover